doc/release-notes.md

```diff
@@ -78,6 +78,10 @@ New RPCs
 New settings
 ------------
 
+- RPC Whitelist system. It can give certain RPC users permissions to only some RPC calls.
+It can be set with two command line arguments (`rpcwhitelist` and `rpcwhitelistdefault`). (#12763)
+Keep in mind that due to UniValue CVE-2019-18936 segfaults could occur if the user gives too much input.
```

Comment:
These are 0.20.0 release notes.
Fixing this is a matter of pulling in a subtree, and it should definitely be fixed before the release! (see #17742)

Reply:
@laanwj I agree, will revert

Reply:
Done